src/reducers/app/thunks.ts

```diff
@@ -459,7 +460,45 @@ export const initLoad = (platformParam = '', networkParam = '', entityParam = ''
   try {
     const { entity, query } = TezosConseilClient.getEntityQueryForId(idParam);
     const items = await executeEntityQuery(serverInfo, platform, network, entity, query);
-    responseOpenModal[2] = { ...responseOpenModal[2], entity, items }
+    let subItems: any[] = []
+
+    if (entity === 'blocks') {
```

Comment:
this if should be avoided, we go through the trouble of dynamically loading the template, we should do something similar with the data methods.